integration_tests/CMakeLists.txt

```
@@ -401,6 +401,7 @@ RUN(NAME format_02 LABELS gfortran)
 RUN(NAME format_04 LABELS gfortran llvm)
 RUN(NAME format_05 LABELS gfortran)
 RUN(NAME format_06 LABELS gfortran llvm)
+RUN(NAME format_07 LABELS gfortran)
```

Should this enable LLVM?
```suggestion
RUN(NAME format_07 LABELS gfortran llvm)
```

---

I am just going to commit it and see if it passes tests.